core/tests/puppeteer-acceptance-tests/specs/blog-admin/assign-roles-to-users-and-change-tag-properties.spec.ts

```diff
@@ -60,6 +60,10 @@ describe('Blog Admin', function () {
         BLOG_RIGHTS.BLOG_ADMIN
       );
       await superAdmin.expectUserToHaveRole('guestUsr1', ROLES.BLOG_ADMIN);
+      await blogAdmin.expectScreenshotToMatch(
+        'expectUserToHaveRole_guestUsr1',
+        __dirname
```

Comment:
@imchristie Is there a way to avoid passing `__dirname` each time? It seems like boilerplate.

Reply:
Unfortunately no. I need to pass the `__dirname` in order to create/specify the folder name and location in the `expectScreenshotToMatch` function.